### PR TITLE
Chore/160028517 seeder improvement

### PR DESCRIPTION
**Pull request by DylanOrsh**
Added functionality to seeder.js to make a dummy game w/ 3 releases, as well as two groups each with a single added non-admin user.

Also fixed what I'm assuming is a typo in that model file.

### CODE REVIEW

File: seed.js [hunk 1]
```diff
@@ -6,37 +6,175 @@ process.env.OUTPUT_LOG = './log.txt';
...
+    groups: [adminGroup._id]
+  });
+  console.log(`User admin created with password ${passwordAdmin}`);
+  return admin.save();
+}
+
+async function makeGame(){
+  console.log('Creating example game with releases...');
+  //set your values as desired
+  var gameTitle = '';
```
Comment (SmithAF) [inline seed.js]: Is there any to declaring these variables? Could would we not just set the properties in the object to  be empty string?

  var gameParams = {
    title: '',
    slug: '',
    repository: '',
    location: '',
    description: 'A placeholder game for testing.'
  };

Comment (SmithAF) [reply seed.js]: Also we should avoid using var. I see no reason why we can't use const

File: seed.js [hunk 2]
```diff
@@ -6,37 +6,162 @@ process.env.OUTPUT_LOG = './log.txt';
...
-
 var app = require('express')();
 
 // stub out the schema
 require('./app/helpers/database')(app);
 
 // create an admin user
-var access = require('./app/helpers/access');
 var User = require('./app/models/user');
+var Game = require('./app/models/game');
```
Comment (SmithAF) [inline seed.js]: For my own sanity can we replace all the vars with const/let?

File: seed.js [hunk 3]
```diff
@@ -1,42 +1,167 @@
...
 // create an admin user
-var access = require('./app/helpers/access');
-var User = require('./app/models/user');
+const User = require('./app/models/user');
+const Game = require('./app/models/game');
+const Group = require('./app/models/group');
+const uuid = require('uuid/v1');
+const Release = require('./app/models/release');
+
+function makeRandomString(length){
```
Comment (chipbell4) [inline seed.js]: 👍

File: seed.js [hunk 4]
```diff
@@ -1,42 +1,167 @@
...
+  });
+  return random;
+}
 
-console.log('Creating admin user...');
-User.createUser(
-  {
+async function makeDummyData(){
+  let admin = await makeAdmin();
+  // let game = await makeGame();
```
Comment (chipbell4) [inline seed.js]: Looks like this is commented out. Any particular reason?

Comment (DylanOrsh) [reply seed.js]: Its a mistake from my local testing, let me clean this up real uick

**Pull request merged**